src/client/vue/pages/PagePlayRemote.vue

```diff
@@ -245,6 +245,24 @@ const initGameView = async () => {
     }
 };
 
+const makeTitle = (gameClient: HostedGameClient) => {
+    const players = gameClient.getPlayers();
+    const { state } = gameClient.getHostedGame();
+    if (players.length < 2 && 'created' === state)
+        return `Waiting for an opponent: hosted by ${players[0].pseudo}`;
+    const pairing = players.map(p => p.pseudo).join(' VS ');
```

Comment:
To prevent showing "3523 VS 9543" for guests, you can use `pseudoString()` (I know it was already missing in some parts written before this function existed)

``` diff
+import { pseudoString } from '../../../shared/app/pseudoUtils';
```

```suggestion
        return `Waiting for an opponent: hosted by ${pseudoString(players[0], 'pseudo')}`;
    const pairing = players.map(p => pseudoString(p, 'pseudo')).join(' VS ');
```